chain-network/proto/watch.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+
+import "types.proto";
+
+package iohk.chain.watch;
+
+message BlockSubscriptionRequest {}
+
+message TipSubscriptionRequest {}
+
+message MempoolSubscriptionRequest {}
+
+message SyncMultiverseRequest {
+  // Block IDs to start synchronization from.
+  //
+  // The client should include the ID of the latest block at or below the
+  // stability depth, as well as tips of any branches known to it from
+  // previously synchronized node state.
```

Comment:
the idea is to get all branches, providing the ones that you know of was mostly to allow optimizations, that is, not sending blocks twice. so.. no?

Reply:
Ok, my confusion is around "ID of the latest block at or below the stability depth". I guess I should read "below the stability depth" as blocks which have a depth greater than the stability one, right?

Reply:
Ok got it thanks, I find the wording a bit confusing though. For example [here](https://github.com/input-output-hk/jormungandr/blob/3c3c1c48b1e557c1538a80c2c2d596ead482278c/jormungandr/src/blockchain/storage.rs#L290) "below the stability depth" conveys the opposite meaning

Reply:
Ehmm, yeah... That's confusing indeed, I find it's usage here more intuitive though, but should probably change it when Mikhail's back (since I think he wrote this part here).